NewEvaluation.css

```diff
@@ -68,3 +92,107 @@ body{
   	width:100%;  background-color: #dfdfdf; text-align: center; vertical-align: center; padding-bottom: 1em;
   	color:#828384;
 }
+
+
+.legend-box legend{
+	color:#03a8f3;
+	font-size: 18px;
+	font-weight: 500;
+	padding-left: 1em;
+	padding-right: 1em;
+}
+
+.legend-box li{
+	float:left;
+	width: 33%;
+	padding-bottom: 10px;
+}
+.legend-box select{
+	width: 40%;
+	padding: 3px;
+}
+.legend-box label{
+	display: inline-block;
+	width: 49%;
+}
+
+
+
+
+#butonu{
+	color:red;
+}
+
+#extra{
```

Comment:
Precisely! That is a good general rule of thumb. Naturally, it will depend from case to case, but most often, that would be the way to go.